src/main/scala/rocket/Rocket.scala

```diff
@@ -642,7 +642,7 @@ class Rocket(implicit p: Parameters) extends CoreModule()(p)
   csr.io.counters foreach { c => c.inc := RegNext(perfEvents.evaluate(c.eventSel)) }
 
   if (enableCommitLog) {
-    val pc = Wire(SInt(width=xLen))
+    val pc = Wire(UInt(width=xLen))
     pc := wb_reg_pc
```

Comment:
I think this should actually be

    val pc = wb_reg_pc.sextTo(xLen)

because the pc may be compressed for certain VM configurations, but in this case the MSB is a sign bit.  The Wire is not necessary.  Can you confirm this works & update the patch?